### PR TITLE
Add setup.py

### PR DESCRIPTION
**Pull request by naglepuff**

### CODE REVIEW

**Comment by jcfr** [comment]
For new package, I suggest to provide a `pyproject.toml` [^1] instead of legacy `setup.py`.

[^1]: https://pip.pypa.io/en/stable/reference/build-system/

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(
+    name='Image DePHI',
```
Comment (brianhelba) [inline setup.py:4]:
```suggestion
    name='ImageDePHI',
```
This shouldn't have spaces, since it's used to build the package file name.

**Review by brianhelba — CHANGES_REQUESTED** [review]
A change to the package name is needed.

**Comment by brianhelba** [comment]
We can discuss `pyproject.toml` in #21. I think this PR is fine using the legacy mechanism.

**Review by brianhelba — APPROVED** [review]

**Pull request merged**